infra/app/envs/dev/main.tf

```diff
@@ -6,7 +6,7 @@ locals {
   # The prefix key/value pair is used for Terraform Workspaces, which is useful for projects with multiple infrastructure developers.
   # By default, Terraform creates a workspace named “default.” If a non-default workspace is not created this prefix will equal “default”, 
   # if you choose not to use workspaces set this value to "dev" 
-  prefix = terraform.workspace
+  prefix = "dev"
```

Comment:
@aplybeah Could we use workspaces? I think it would be helpful to me for this project to have access that.